clusterloader2/testing/load/config.yaml

```diff
@@ -20,6 +20,7 @@
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
 {{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS true}}
 {{$ENABLE_NETWORKPOLICIES := DefaultParam .CL2_ENABLE_NETWORKPOLICIES false}}
+{{$ENABLE_DNSTESTS := DefaultParam .CL2_ENABLE_DNSTESTS true}}
```

Comment:
Before submitting please change this to false by default. We want to enable this in our tests gradually.

Reply:
Yes, i plan to change it before submitting. This was only to get a presubmit test to verify that the config is correct.